src/lib/datepicker/index.ts

```diff
@@ -20,8 +20,27 @@ export * from './year-view';
 
 
 @NgModule({
-  imports: [CommonModule, DatetimeModule, MdDialogModule, OverlayModule],
-  exports: [MdCalendar, MdCalendarTable, MdDatepicker, MdDatepickerInput, MdMonthView, MdYearView],
-  declarations: [MdCalendar, MdCalendarTable, MdDatepicker, MdDatepickerInput, MdMonthView, MdYearView],
+  imports: [
+    CommonModule,
+    DatetimeModule,
+    MdDialogModule,
+    OverlayModule
+  ],
+  exports: [
+    MdCalendar,
+    MdCalendarTable,
```

Comment:
Do you intend for the CalendarTable, MonthView, etc. to be used externally? Seems like you can remove them from the exports list.